docs/source/workflow_inputs.rst

```diff
@@ -1529,6 +1529,7 @@ Each electric resistance heating system is entered as a ``/HPXML/Building/Buildi
   ``HeatingCapacity``                                            double   Btu/hr  >= 0                No        autosized [#]_  Heating output capacity
   ``AnnualHeatingEfficiency[Units="Percent"]/Value``             double   frac    > 0, <= 1           Yes                       Efficiency
   ``FractionHeatLoadServed``                                     double   frac    >= 0, <= 1 [#]_     See [#]_                  Fraction of heating load served
+  ``extension/HeatingAutosizingFactor``                          double   frac    >= 0                No        1.0             Heating autosizing scaling factor
```

Comment:
EPvalidator.xml says it must be greater than zero, so these should all be changed to `> 0`.